packages/myst-to-tex/src/index.ts

```diff
@@ -420,6 +471,18 @@ class TexGlossarySerializer {
     );
     return [usepackage, makeglossaries].concat(entries);
   }
+
+  private createAcronymDirectives(acronymDefinitions: Record<string, [string, string]>): string[] {
+    const directives = Object.keys(acronymDefinitions).map((k) => ({
+      key: k,
+      acronym: acronymDefinitions[k][0],
+      expansion: acronymDefinitions[k][1],
+    }));
+
+    return directives.map(
```

Comment:
Good catch :) So a few points:
- When creating the first change, I am quite sure I did use that command: `\usepackage[acronym]{glossaries}` because I saw the docs in overleaf as well and learnt about that special case.
- I believe that when merging the changes for glossaries, the command got overwritten :)
- I have been running since yesterday: tests and the application e2e with just `\usepackage{glossaries}` and it has been working just fine. My guess? That optional argument in the TeX directive used to be necessary in the past, not it is not anymore :)